src/app/core/nav-menu/components/license/license.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'cs-license',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="license">
+      {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_1' | translate: { year: getCurrentYear() } }}
+      <a href="https://bitworks.software/" target="_blank">
+        {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_2' | translate }}
+      </a>
+      <br>
+      {{ 'NAVIGATION_SIDEBAR.LICENSE.LICENSE_3' | translate }}
+    </div>
```

Comment:
It is not a good way to format content using `<br>` and separate translations for each line, because text can have different order and formatting for different languages.

For the lines, I suggest using wrappers like `<p>` or `<div>` instead of `<br>`.

As far as I know, you can put links and markup right inside of translation and pass all required data using translation params (year, link etc.). Isn't it?